src/Main.js

```diff
@@ -4,30 +4,31 @@ import ExtendedView from './ExtendedView'
 import List from './List'
 import movieData from './mockData'
 import './Main.css'
+import {getAllMovies, fetchSingleMoviePoster} from './apiCalls'
 
 class Main extends React.Component {
   constructor() {
     super();
     this.state = {
       movieData: [],
+      singleMovie: [],
```

Comment:
Nice work here adding more properties of state

Reply:
Why is `singleMovie` an empty array? If a variable is not going to be an array later, don't use an empty one as a placeholder. `null` would be better.